jkube-kit/enricher/generic/src/test/java/org/eclipse/jkube/enricher/generic/ProjectLabelEnricherTest.java

```diff
@@ -31,167 +30,115 @@
 import org.eclipse.jkube.kit.config.resource.GroupArtifactVersion;
 import org.eclipse.jkube.kit.config.resource.PlatformMode;
 import org.eclipse.jkube.kit.enricher.api.JKubeEnricherContext;
-import org.junit.Before;
-import org.junit.Test;
 import io.fabric8.kubernetes.api.model.KubernetesList;
 import io.fabric8.kubernetes.api.model.KubernetesListBuilder;
 import io.fabric8.kubernetes.api.model.apps.Deployment;
 import io.fabric8.kubernetes.api.model.apps.DeploymentBuilder;
 import io.fabric8.openshift.api.model.DeploymentConfigBuilder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
 
 /**
  * Test label generation.
  *
  * @author Tue Dissing
  */
-public class ProjectLabelEnricherTest {
-  private JKubeEnricherContext context;
+class ProjectLabelEnricherTest {
 
   private Properties properties;
   private ProjectLabelEnricher projectLabelEnricher;
 
-  @Before
-  public void setupExpectations() {
-    context = mock(JKubeEnricherContext.class, RETURNS_DEEP_STUBS);
+  @BeforeEach
+  void setupExpectations() {
+    JKubeEnricherContext context = mock(JKubeEnricherContext.class, RETURNS_DEEP_STUBS);
     projectLabelEnricher = new ProjectLabelEnricher(context);
     properties = new Properties();
     when(context.getProperties()).thenReturn(properties);
     when(context.getGav()).thenReturn(new GroupArtifactVersion("groupId", "artifactId", "version"));
   }
 
   @Test
-  public void testCustomAppName() {
-    // Setup
+  void create_customAppName() {
```

Comment:
You need to create an additional test for DeploymentConfig, the test you refactored was testing both Deployment and DeploymentConfig